src/play/modules/s3blobs/S3Blobs.java

```diff
@@ -32,7 +38,11 @@ public void onApplicationStart() {
         String accessKey = Play.configuration.getProperty("aws.access.key");
         String secretKey = Play.configuration.getProperty("aws.secret.key");
         AWSCredentials awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
-        S3Blob.s3Client = new AmazonS3Client(awsCredentials);
+
+        S3Blob.s3Client = AmazonS3ClientBuilder.standard()
+                .withRegion(Regions.EU_WEST_2)
```

Comment:
Seems like we should externalize this via config.

Reply:
Agreed, will make that change

Reply:
Maybe there was a default before?  Do we need to override it?  I can't remember what the region is used for in this context, or can it figure it out?

Reply:
A client created via the builders must have a region that is defined either explicitly (i.e. by calling withRegion) or as part of the DefaultAwsRegionProviderChain. If the builder can’t determine the region for a client, an SdkClientException is thrown

Reply:
Ah cool.  So seems like we should require it in the config.  Thanks!